pritunl.conf

```diff
@@ -0,0 +1,10 @@
+{
+    "host_id": "ffd7a38db01042cf806760c211ff4a41",
```

Comment:
what's the host ID ?

Reply:
It will be auto-generated on pritunl start, we will omit it from conf file.